main.cc

```diff
@@ -1755,6 +1755,24 @@ sharded<locator::shared_token_metadata> token_metadata;
 
```

Comment:
The commit-log is confusing: it says the stream-manager is currently initialized *before* the storage-service, but this is what the patch seems to be doing, so it appears that previously to the patch, stream manager was started *after* storage service.